src/aat/resources/features/F-063/S-561.td.json

```diff
@@ -0,0 +1,34 @@
+{
+  "title": "must return 404 when case type does not exist",
+
+  "_guid_": "S-561",
+  "_extends_": "F-063_Base_Test_Data",
+
+  "specs": [
+    "an active profile in CCD",
+    "contains a case type that does not exist",
+    "contains a HTTP 404 Not Found"
+  ],
+
+  "request": {
+    "pathVariables": {
+      "ctid": "abc"
```

Comment:
as 'abc' might be valid can we put 'non-existing case type ID' as suggested in the past to me as well

Reply:
updated to "non-existing case type" 👍